packages/contracts/contracts/tcr/CivilTCR.sol

```diff
@@ -84,23 +85,26 @@ contract CivilTCR is RestrictedAddressRegistry {
 
   /**
   @notice Requests an appeal for a listing that has been successfully challenged (and had state updated)
-          Must be initialized and within correct application period 
-          (0 < appeal.requestAppealPhaseExpiry < now) 
-          and not already requested
-  @param listingAddress address of listing that has been successfully challenged. Caller must be owner of listing.
+  In order to request appeal, the following conditions must be met:
+  - voting for challenge has ended
```

Comment:
The point of using natspec comments is so that our documentation can be autogenerated. That's the main reason we spent time getting the generated docs working. I think it's fine (and good) to do this documentation directly above the code it's documenting.